app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -106,4 +118,31 @@ class ConversationListViewModel @Inject constructor(
 
     }
 
+    private fun List<Conversation>.toGeneralConversationList() = map { conversation ->
+        if (isPrivateChat(conversation)) {
+            GeneralConversation(
+                ConversationType.PrivateConversation(
+                    userInfo = UserInfo(),
+                    conversationInfo = ConversationInfo(
+                        name = "Some private chat",
+                        membership = Membership.None,
+                        isLegalHold = true
+                    ),
+                    conversationsId = conversation.id
+                )
+            )
+        } else {
+            GeneralConversation(
+                ConversationType.GroupConversation(
+                    groupColorValue = 0xFFFF0000,
```

Comment:
This color could be extracted or reuse one from `WireColorPalette` class

Reply:
Yes, we should use `MaterialTheme.wireColorScheme` instead